clang/test/DebugInfo/CXX/callsite-derived.cpp

```diff
@@ -0,0 +1,77 @@
+// RUN: %clang_cc1 -triple=x86_64-linux -disable-llvm-passes -emit-llvm \
+// RUN:            -debug-info-kind=constructor -dwarf-version=5 -O1 %s \
+// RUN:            -o - | FileCheck %s -check-prefix CHECK-DERIVED
+
+// Simple base and derived class with virtual and static methods:
+// We check for a generated 'call_target' for:
+// - 'one', 'two' and 'three'.
+
+class CBase {
+public:
+  virtual void one(bool Flag) {}
+  virtual void two(int P1, char P2) {}
+  static void three();
+};
+
+void CBase::three() {
+}
+void bar(CBase *Base);
+
+void foo(CBase *Base) {
+  CBase::three();
+}
+
+class CDerived : public CBase {
+public:
```

Comment:
probably use a `struct` (here and elsewhere) since it's public inheritance and members by default